src/Datadog.Trace/Tags.cs

```diff
@@ -318,6 +318,26 @@ public static class Tags
         /// </summary>
         public const string Measured = "_dd.measured";
 
+        /// <summary>
+        /// The name of the Msmq command the message was published to.
+        /// </summary>
+        public const string MsmqCommand = "msmq.command";
+
+        /// <summary>
+        /// Is the msmq queue supporting transactional messages
+        /// </summary>
+        public const string MsmqIsTransactionalQueue = "msmq.is_transactional_queue";
+
+        /// <summary>
+        /// The name of the Msmq queue the message was published to, containing host name and path.
+        /// </summary>
+        public const string MsmqQueueUniqueName = "msmq.queue_unique_name";
+
+        /// <summary>
+        /// A boolean indicating if it's part of a transaction.
+        /// </summary>
+        public const string MsmqMessageWithTransaction = "msmq.message_with_transaction";
```

Comment:
What do you think of the following tag names?
- `msmq.queue.name`
- `msmq.queue.transactional`
- `msmq.message.transactional`